topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -1911,18 +1918,14 @@ We just generated the first input for **goseq**. As second input for **goseq** w
 >    {% snippet faqs/galaxy/histories_copy_dataset.md %}
 > </div>
 >
-{: .hands_on}
-
-> ### {% icon hands_on %} Hands-on: Adapt the gene length to goseq
->
-> 1. {% tool [Change Case](ChangeCase) %} with the following parameters:
+> 2. {% tool [Change Case](ChangeCase) %} with the following parameters:
```

Comment:
I divided the hands on in 2 parts because if we merge the numbering does not render correctly, we get:
![image](https://user-images.githubusercontent.com/30404086/167106434-4cb52e49-6f2c-431a-8a6a-8aee24610df5.png)
![image](https://user-images.githubusercontent.com/30404086/167106519-064d9577-7849-4511-89fb-6201930fd1e3.png)

Reply:
(It is not a big deal, just to explain you why I did that)

Reply:
I do not understand why it is doing that... It should not. I will have a look

Reply:
Yes because there is this `<div>` section... But it is not a big deal...